### PR TITLE
Democracy audit checks

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to ensure that democracy can take place on mainnet.

Part of #255 

# Discussion
- Mainnet: Added two additional call filters so that preimage and scheduling pallets can be called.
- Rococo updated  governance times (and bumped spec version)
- Local updated governance times
- Local added additional council member

### CODE REVIEW

File: runtime/common/src/constants.rs [hunk 1]
```diff
@@ -182,11 +182,11 @@ parameter_types! {
...
-	pub EnactmentPeriod: BlockNumber = prod_or_testnet_or_local_or_env!(8 * DAYS, 28 * DAYS, 28 * DAYS, "FRQCY_ENACTMENT_PERIOD");
-	pub CooloffPeriod: BlockNumber = prod_or_testnet_or_local_or_env!(7 * DAYS, 7 * DAYS, 7 * DAYS, "FRQCY_COOLOFF_PERIOD");
+	pub LaunchPeriod: BlockNumber = prod_or_testnet_or_local_or_env!(7 * DAYS, 1 * DAYS, 5 * MINUTES, "FRQCY_LAUNCH_PERIOD");
+	pub VotingPeriod: BlockNumber = prod_or_testnet_or_local_or_env!(7 * DAYS, 1 * DAYS, 5 * MINUTES, "FRQCY_VOTING_PERIOD");
+	pub FastTrackVotingPeriod: BlockNumber = prod_or_testnet_or_local_or_env!(3 * HOURS, 30 * MINUTES, 5 * MINUTES, "FRQCY_FAST_TRACK_VOTING_PERIOD");
+	pub EnactmentPeriod: BlockNumber = prod_or_testnet_or_local_or_env!(8 * DAYS, 30 * HOURS, 10 * MINUTES, "FRQCY_ENACTMENT_PERIOD");
+	pub CooloffPeriod: BlockNumber = prod_or_testnet_or_local_or_env!(7 * DAYS, 1 * DAYS, 5 * MINUTES, "FRQCY_COOLOFF_PERIOD");
 	pub MinimumDeposit: Balance = prod_or_testnet_or_local_or_env!(currency::deposit(5, 0), 100 * currency::deposit(5, 0), 100 * currency::deposit(5, 0), "FRQCY_MINIMUM_DEPOSIT");
```
Comment (wilwade) [inline runtime/common/src/constants.rs:190]: None of these changed for mainnet.

File: runtime/frequency/src/lib.rs [hunk 2]
```diff
@@ -79,6 +79,8 @@ impl Contains<Call> for BaseCallFilter {
 			Call::Council(..) => true,
 			Call::Democracy(..) => true,
 			Call::Session(..) => true,
+			Call::Preimage(..) => true,
+			Call::Scheduler(..) => true,
```
Comment (wilwade) [inline runtime/frequency/src/lib.rs:83]: Mainnet change!

**Review by saraswatpuneet — APPROVED** [review]
:100:

**Review by enddynayn — APPROVED** [review]

File: node/service/src/chain_spec/frequency_local.rs [hunk 3]
```diff
@@ -71,6 +71,7 @@ pub fn local_testnet_config() -> ChainSpec {
 				vec![
 					get_account_id_from_seed::<sr25519::Public>("Alice"),
 					get_account_id_from_seed::<sr25519::Public>("Charlie"),
+					get_account_id_from_seed::<sr25519::Public>("Eve"),
```
[the remark this answers is not in the document]
Comment (wilwade) [reply node/service/src/chain_spec/frequency_local.rs:74]: No. I was able to fasttrack once I was able to setup an external proposal that didn't require "all" the council members to agree.

[the remark this answers is not in the document]
Comment (wilwade) [reply node/service/src/chain_spec/frequency_local.rs:74]: State of the chain I was able to do it on:

image
image

**Pull request merged**